helmfile/values/networkpolicy-common.yaml.gotmpl

```
@@ -32,3 +34,12 @@ kured:
 
 velero:
   enabled: {{ and .Values.velero.enabled .Values.networkPolicies.velero.enabled }}
+
+certManager:
+  enabled: {{ .Values.networkPolicies.certManager.enabled }}
+  namespaces: {{ toYaml .Values.networkPolicies.certManager.namespaces | nindent 4 }}
+    {{- if eq .Environment.Name "workload_cluster" }}
+    {{- toYaml .Values.user.namespaces | nindent 4 }}
+    {{- end }}
```

```suggestion
  namespaces:
    {{- if eq .Environment.Name "workload_cluster" }}
    {{- concat .Values.user.namespaces .Values.networkPolicies.certManager.namespaces | toYaml | nindent 4 }}
    {{- else }}
    {{- toYaml .Values.networkPolicies.certManager.namespaces | nindent 4 }}
    {{- end }}
```
It didn't render properly if `networkPolicies.certManager.namespaces` was empty.